contrib/terraform/openstack/kubespray.tf

```diff
@@ -1,3 +1,8 @@
+provider "openstack" {
+  version = "1.5.0"
+  use_octavia = "true"
```

Comment:
Would it make more sense to use a boolean here? https://www.terraform.io/docs/configuration/variables.html#booleans

Reply:
Sure.  The terraform provider expects a boolean here, will update.